tests/sound/sound.c

```diff
@@ -0,0 +1,278 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/* Sound request */
+enum {
+    INIT_SOUND,
+    SHUTDOWN_SOUND,
+    PLAY_MUSIC,
+    PLAY_SFX,
+    SET_MUSIC_VOLUME,
+    STOP_MUSIC,
+};
+
+typedef struct {
+    void *data;
+    int size;
+} musicinfo_t;
+
+typedef struct sfxinfo_struct {
```

Comment:
Simply remove `sfxinfo_struct` as an anonymous structure.

Reply:
To avoid code duplication, let's move the structure definitions and the function prototype to the header, `src/mus2midi.h`.